src/components/Repository.js

```diff
@@ -83,6 +83,17 @@ function Repository({match}) {
                 </a>
               </em>
             </small>
+            <div style={{marginTop: 20}}>
+              {repository && repository.languages.nodes.map((language, key) => (
+                <span key={key}>
+                  <span style={{color: language.color, fontSize: 25, margin: 0, verticalAlign: "middle"}}>•</span>
```

Comment:
```suggestion
                  <span style={{color: language.color, fontSize: 25, margin: 0, verticalAlign: "sub"}}>•</span>
```

This is this fix to the alignment issue

Reply:
I am noticing the octicon alignment has weird results if the incorrect attribute is used.